### PR TITLE
Papa 7 deploy sheets

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated mooring deploy sheets for Papa 7:
GP02HYPM_Deploy
GP03FLMA_Deploy
GP03FLMB_Deploy
*Also removed 'check deployment depth' note for previous deployment on Papa 6.  Depths were confirmed but note never removed.

**Update: 8/30/19 - added Glider deploy sheets:
GP05MOAS-GL537_Deploy
GP05MOAS-PG575_Deploy

- New glider sensor cals will be on separate PR

- Will need to update all after deployment w/ deploy date/time, recover date/time, lat/lon, depth.

### CODE REVIEW

File: deployment/GP05MOAS-PG575_Deploy.csv [hunk 1]
```diff
@@ -5,4 +5,11 @@ RB-16-05,,RB-16-05,,GP05MOAS-PG575-02-DOSTAM000,1,1,2016-06-27T18:58:00,2016-07-
...
+SKQ201920S,,,,GP05MOAS-PG575-00-ENG000000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGCON-GLDRGP-00575,,,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-01-CTDGVM000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-CTDGVM-09287,,,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-02-DOSTAM000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-DOSTAM-00414,,,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-03-FLORTM000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTM-03815,,,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-04-FLORTO000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTO-01332,,,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-05-NUTNRM000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-NUTNRM-00711,,,,200,200,
+SKQ201920S,,,,GP05MOAS-PG575-06-PARADM000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-PARADM-50200,,,,200,200,
```
Comment (cdobs) [inline deployment/GP05MOAS-PG575_Deploy.csv]: Looks good

File: deployment/GP05MOAS-GL537_Deploy.csv [hunk 2]
```diff
@@ -2,4 +2,8 @@ CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentN
...
 TN-323,,RB-16-05,,GP05MOAS-GL537-02-DOSTAM000,1,1,2015-06-02T11:05:00,2016-06-27T05:02:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00400,50.17548,-144.55345,,1000,1000,
-TN-323,,RB-16-05,,GP05MOAS-GL537-04-CTDGVM000,1,1,2015-06-02T11:05:00,2016-06-27T05:02:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,50.17548,-144.55345,,1000,1000,
\ No newline at end of file
+TN-323,,RB-16-05,,GP05MOAS-GL537-04-CTDGVM000,1,1,2015-06-02T11:05:00,2016-06-27T05:02:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,50.17548,-144.55345,,1000,1000,
+SKQ201920S,,,,GP05MOAS-GL537-00-ENG000000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGCON-GLDROO-00537,,,,1000,1000,
+SKQ201920S,,,,GP05MOAS-GL537-01-FLORDM000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-FLORDM-03582,,,,1000,1000,
+SKQ201920S,,,,GP05MOAS-GL537-02-DOSTAM000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00401,,,,1000,1000,
+SKQ201920S,,,,GP05MOAS-GL537-04-CTDGVM000,2,1,2019-09-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,,,,1000,1000,
```
Comment (cdobs) [inline deployment/GP05MOAS-GL537_Deploy.csv]: Looks good

File: deployment/GP02HYPM_Deploy.csv [hunk 3]
```diff
@@ -69,17 +69,31 @@ SR17-10,,SR1811,,GP02HYPM-WFP03-01-FLORDL000,5,1,2017-07-15T21:25:00,2018-07-26T
...
+SKQ201920S,,,,GP02HYPM-WFP02-01-FLORDL000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-FLORDL-02934,,,,,,
+SKQ201920S,,,,GP02HYPM-WFP02-03-DOSTAL000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-DOSTAL-01105,,,,,,
+SKQ201920S,,,,GP02HYPM-WFP02-04-CTDPFL000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-CTDPFL-00115,,,,,,
+SKQ201920S,,,,GP02HYPM-WFP02-05-VEL3DL000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,,,,,,
+SKQ201920S,,,,GP02HYPM-WFP03-00-WFPENG000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,,,,,,
+SKQ201920S,,,,GP02HYPM-WFP03-01-FLORDL000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,,,,,,
+SKQ201920S,,,,GP02HYPM-WFP03-03-DOSTAL000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,,,,,,
+SKQ201920S,,,,GP02HYPM-WFP03-04-CTDPFL000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,,,,,,
+SKQ201920S,,,,GP02HYPM-WFP03-05-VEL3DL000,7,1,2019-09-20T00:00:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,,,,,,
```
Comment (cdobs) [inline deployment/GP02HYPM_Deploy.csv]: Looks good for pre-deployment status

File: deployment/GP03FLMA_Deploy.csv [hunk 4]
```diff
@@ -89,21 +89,39 @@ SR17-10,,SR1811,,GP03FLMA-RIS01-00-SIOENG000,5,1,2017-07-13T23:00:00,2018-07-27T
...
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG046,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11593,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG047,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11535,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG048,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-12308,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOH049,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOH-11683,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOH050,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOH-10264,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOH051,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOH-12389,,,,,,
+SKQ201920S,,,,GP03FLMA-RIS01-00-SIOENG000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGCON-SIOCON-14876,,,,,,
+SKQ201920S,,,,GP03FLMA-RIS01-03-DOSTAD000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-DOSTAD-00379,,,,,,
+SKQ201920S,,,,GP03FLMA-RIS01-04-PHSENF000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-PHSENF-P0103,,,,,,
+SKQ201920S,,,,GP03FLMA-RIS01-05-FLORTD000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-FLORTD-01122,,,,,,
```
Comment (cdobs) [inline deployment/GP03FLMA_Deploy.csv]: Looks good

File: deployment/GP03FLMB_Deploy.csv [hunk 5]
```diff
@@ -89,21 +89,39 @@ SR17-10,,SR1811,,GP03FLMB-RIS01-00-SIOENG000,5,1,2017-07-14T22:49:00,2018-07-25T
...
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG066,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-12406,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG067,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-11605,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG068,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-12310,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOH069,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOH-12388,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOH070,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOH-12618,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOH071,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOH-10228,,,,,,
+SKQ201920S,,,,GP03FLMB-RIS01-00-SIOENG000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGCON-SIOCON-13284,,,,,,
+SKQ201920S,,,,GP03FLMB-RIS01-03-DOSTAD000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-DOSTAD-00519,,,,,,
+SKQ201920S,,,,GP03FLMB-RIS01-04-PHSENF000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-PHSENF-P0147,,,,,,
+SKQ201920S,,,,GP03FLMB-RIS01-05-FLORTD000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-FLORTD-01298,,,,,,
```
Comment (cdobs) [inline deployment/GP03FLMB_Deploy.csv]: Looks good for pre-deployment status

File: deployment/GP02HYPM_Deploy.csv (hunk 3 above)
Comment (pbrickley) [reply deployment/GP02HYPM_Deploy.csv]: Looks good.

File: deployment/GP05MOAS-GL537_Deploy.csv (hunk 2 above)
Comment (pbrickley) [reply deployment/GP05MOAS-GL537_Deploy.csv]: Good.

File: deployment/GP05MOAS-PG575_Deploy.csv (hunk 1 above)
Comment (pbrickley) [reply deployment/GP05MOAS-PG575_Deploy.csv]: good

File: deployment/GP03FLMA_Deploy.csv (hunk 4 above)
Comment (pbrickley) [reply deployment/GP03FLMA_Deploy.csv]: Are the lat lon correct?  Surveyed anchor locations are (decimal minutes):
HYPM-7 50 06.911 N 144 54.967 W
FLMA-7 50 01.362 N 144 21.647 W
FLMB-7 50 22.533 N 144 30.870 W

File: deployment/GP03FLMB_Deploy.csv (hunk 5 above)
Comment (pbrickley) [reply deployment/GP03FLMB_Deploy.csv]: Are the lat lon correct? Surveyed anchor locations are (decimal minutes):
HYPM-7 50 06.911 N 144 54.967 W
FLMA-7 50 01.362 N 144 21.647 W
FLMB-7 50 22.533 N 144 30.870 W

**Pull request merged**